pkg/controller/webhooks/postgresql.go

```diff
@@ -97,10 +131,9 @@ func (p *PostgreSQLWebhookHandler) ValidateDelete(ctx context.Context, obj runti
 }
 
 // checkPostgreSQLQuotas will read the plan if it's set and then check if any other size parameters are overwriten
-func (p *PostgreSQLWebhookHandler) checkPostgreSQLQuotas(ctx context.Context, pg *vshnv1.VSHNPostgreSQL, checkNamespaceQuota bool) *apierrors.StatusError {
+func (p *PostgreSQLWebhookHandler) checkPostgreSQLQuotas(ctx context.Context, pg *vshnv1.VSHNPostgreSQL, checkNamespaceQuota bool, allErrs *field.ErrorList) *apierrors.StatusError {
```

Comment:
I'm fine if you are passing the list here as a pointer though you have to get rid of `*apierrors.StatusError`. There should be a bit of refactoring otherwise it's a mess.